build/package/debian/control

```diff
@@ -1,14 +1,14 @@
 Source: nginx-asg-sync
-Section: misc
+Section: contrib
```

Comment:
I was following this one https://www.debian.org/doc/debian-policy/ch-archive.html#sections and used `contrib` because we require nginx-plus which is not free. Maybe should be `contrib/misc`?

Reply:
ok. contrib sounds good